### ISSUE TITLE
suggestion extract VLANs and LAG? 

### ISSUE THREAD

**Issue by FlyingToto** [MarkGodwin/tplink-omada-api#47]
anyway to identify VLANs settings when setup by ports? 
(or is this what the Type and Profile refer to?)

example:

Port:  4 - Port4
    Type:        1
    Profile:     All - ☐
    Enabled:     ☑
    Status:      ☑
    PoE:         ☑    0.0W
    Link Speed:  SPEED_100_MBPS (Max: SPEED_1_GBPS)
    Transmitted: 1.3GB
    Received:    95.0MB
    Operation:   switching
    Limit Mode:  OFF
Port:  5 - Port5
    Type:        1
    Profile:     All - ☐
    Enabled:     ☑
    Status:      ☐
    PoE:         ☑    0.0W
    Link Speed:  SPEED_10_MBPS (Max: SPEED_1_GBPS)
    Transmitted: 0.0MB
    Received:    0.0MB
    Operation:   switching
    Limit Mode:  OFF

**Comment by MarkGodwin** [comment]
Yes, I think you can only set VLANs for ports by changing the port profile.
Currently, the CLI doesn't have any support for listing profiles or setting them on a port. If you need this, I can look at adding it.
